src/adapter.ts

```diff
@@ -262,4 +282,21 @@ export default class JestTestAdapter implements TestAdapter {
   private retireAllTests() {
     this.retireEmitter.fire({});
   }
+
+  private logSuite(suiteOrTest?: TestSuiteInfo|TestInfo, depth: number = 0): void {
+    if (suiteOrTest == null) { return; }
```

Comment:
Minor suggestion, just my allergies to type coercion.

```suggestion
    if (_.isNil(suiteOrTest)) { return; }
```